icone_arrow_boards/get_icone.py

```diff
@@ -0,0 +1,53 @@
+import urllib.request
+import datetime
+import os
+import xmltodict
+
+ICONE_USERNAME = os.getenv('ICONE_USERNAME')
+ICONE_FILE_PATH = os.getenv('ICONE_FILE_PATH')
+ICONE_PASSWORD = os.getenv('ICONE_PASSWORD')
+
+print(ICONE_USERNAME, ICONE_FILE_PATH, ICONE_PASSWORD)
```

Comment:
Let's convert these print statements to use the logging framework. Also, do we really want to print out the icone username/password?

Reply:
Removed